dbt/adapters/bigquery/impl.py

```diff
@@ -209,14 +223,22 @@ def check_schema_exists(self, database: str, schema: str) -> bool:
     def get_columns_in_relation(self, relation: BigQueryRelation) -> List[BigQueryColumn]:
         try:
             table = self.connections.get_bq_table(
-                database=relation.database, schema=relation.schema, identifier=relation.identifier
+                database=relation.database,
+                schema=relation.schema,
+                identifier=relation.identifier,
             )
             return self._get_dbt_columns_from_bq_table(table)
 
         except (ValueError, google.cloud.exceptions.NotFound) as e:
             logger.debug("get_columns_in_relation error: {}".format(e))
             return []
 
+    @available.parse(lambda *a, **k: [])
+    def add_time_ingestion_partition_column(self, columns) -> List[BigQueryColumn]:
+        "Add time ingestion partition column to columns list"
+        columns.append(self.Column("_PARTITIONTIME", "TIMESTAMP", None, None))
```

Comment:
```suggestion
        columns.append(self.Column("_PARTITIONTIME", "TIMESTAMP", None, "NULLABLE"))
```

There are four arguments to `Column.__init__` besides `self`: `column`, `dtype`, `fields`, and `mode`. I don't think that `mode` can be `None`, so either pass nothing or pass `"nullable"` I think.

https://github.com/dbt-labs/dbt-bigquery/blob/3d6986984fffb21d82d3365e7cc5d72545f602e6/dbt/adapters/bigquery/column.py#L23-L29

Reply:
Ok good point, I'll try tomorrow, my issue was on an uncommitted change but it's very similar 👍